### PR TITLE
Prepare 0.1.2 release.

### PR DESCRIPTION
**Pull request by aomarks**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Change Log
 
 ## [Unreleased]
-- Added "observer" attribute to @property annotation
-- Added "computed" attribute to @property annotation
-- Fix bug where use of `Polymer.mixinBehaviors` with `@customElement` would throw an exception relating to the definition of the element's "properties" property.
+
+## [0.1.2] - 2018-01-01
+- Added `observer` and `computed` attribute to `@property` decorator.
```
Comment (justinfagnani) [inline CHANGELOG.md]: Maybe: Added `observer` and `computed` properties to the `@property` decorator options.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+- Added `@listen` decorator and `DeclarativeEventListeners` mixin.
+- Fixed bug where use of `Polymer.mixinBehaviors` with `@customElement` would throw an exception relating to the definition of the element's "properties" property.
```
Comment (justinfagnani) [inline CHANGELOG.md]: I think it was with the `@property` decorator, no?

File: CHANGELOG.md (hunk 1 above)
Comment (aomarks) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (aomarks) [reply CHANGELOG.md]: Good point. Or `@computed`. Updated.

**Pull request merged**